src/components/QueryEditor/QueryEditorForm.test.tsx

```diff
@@ -1,10 +1,61 @@
 import { queryTypeOptionToQueryType } from './QueryEditorForm';
 import { queryTypeOptions } from './constants';
 import { XrayQueryType } from '../../types';
+import { TemplateSrv } from '@grafana/runtime';
+import { ScopedVars, VariableModel } from '@grafana/data';
+
+jest.mock('@grafana/runtime', () => {
```

Comment:
is there a way to just mockout getTemplateSrv ? It seems like that's the only bit you're using right?

Reply:
switched it to a jest.SpyOn